Foundation/NSURLResponse.swift

```diff
@@ -233,7 +233,8 @@ open class HTTPURLResponse : URLResponse {
     /// This property is intended to produce readable output.
     override open var description: String {
         var result = "<\(type(of: self)) \(Unmanaged.passUnretained(self).toOpaque())> { URL: \(url!.absoluteString) }{ status: \(statusCode), headers {\n"
-        for(key, value) in allHeaderFields {
+        for(aKey, aValue) in allHeaderFields {
+            let (key,value) = (aKey as! String, aValue as! String) //allHeaderFields is either assigned a [String:String] or [:]
```

Comment:
Yes, though allHeaderFields is always derived from a ```[String:String]``` we can have subclasses storing other types in it. I've updated the PR. Thanks.